resources/fan_control/fan_control.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Fan control
+After=remote-fs.target
+After=syslog.target
+After=nvpmodel.service
```

Comment:
This might fail on xavier nx. Therefore the fan_control will never run. This After can be deleted.

Reply:
Removed in https://github.com/airvolute/dcs-deploy/pull/14/commits/fac4d99a6afe7d6bc713bf795c8fec2999204c9b

Reply:
:D The problem is, it does not work with this fix: fac4d99a6afe7d6bc713bf795c8fec2999204c9b

It only works when highlighted line is there.

Reply:
Hmm, if it does work then we should not break it.... :D